### PR TITLE
Mouseover submenu select

### PR DESCRIPTION
**Pull request by beckyconning**

### CODE REVIEW

**Comment by beckyconning** [comment]
@jonsterling keep in mind the refactoring issue #10

File: src/Halogen/Menu/Component.purs [hunk 1]
```diff
@@ -116,12 +117,28 @@ menuComponent = parentComponent' render eval peek
...
+        [ H.div_
+            [ renderButtonWithMouseoverAction (SelectSubmenu index) menuSubmenu.label ]
+        ]
 
     renderButton :: forall f p. Action f -> String -> HTML p f
     renderButton a label =
       H.button
-        [ E.onClick (\_ -> EH.preventDefault *> EH.stopPropagation $> a unit) ]
+        [ E.onClick (\_ -> EH.preventDefault *> EH.stopPropagation $> a unit)
+        ]
```
Comment (beckyconning) [inline src/Halogen/Menu/Component.purs]: #oops

**Comment by beckyconning** [comment]
by "keep in mind the refactoring issue #10" i mean, i'll refactor it in another pr : )

**Comment by jonsterling** [comment]
The code changes looks good to me!

**Comment by beckyconning** [comment]
Ace thanks @jonsterling

**Pull request merged**